specs/exec-engine.md

```diff
@@ -15,10 +15,12 @@
     - [Pre-Ecotone](#pre-ecotone)
     - [Ecotone L1-Cost fee changes (EIP-4844 DA)](#ecotone-l1-cost-fee-changes-eip-4844-da)
 - [Engine API](#engine-api)
-  - [`engine_forkchoiceUpdatedV2`](#engine_forkchoiceupdatedv2)
-    - [Extended PayloadAttributesV2](#extended-payloadattributesv2)
+  - [`engine_forkchoiceUpdatedV3`](#engine_forkchoiceupdatedv3)
```

Comment:
If v2 is still required for bedrock and canyon then should we add a new section and leave the old section instead?

Reply:
I believe we only need v2 for `newPayload`. `forkchoiceUpdated` and `getPayload` the V3 API's are backwards compatible and we can use them for all blocks.